src/gwt/src/org/rstudio/studio/client/projects/StudioClientProjectConstants_en.properties

```diff
@@ -145,6 +145,10 @@ chooseFileCaption=Choose File
 enableCodeIndexingLabel=Index source files (for code search/navigation)
 chkSpacesForTabLabel=Insert spaces for tab
 tabWidthLabel=Tab width
+useNativePipeOperatorLabel=Use native pipe operator, |> (requires R 4.1+)
+useNativePipeOperatorNeverText=Never (use magrittr pipe)
+useNativePipeOperatorR41Text=Only when using R 4.1+
+useNativePipeOperatorAlwaysText=Always
```

Comment:
in 🇫🇷: 

```
useNativePipeOperatorLabel=Utiliser le pipe natif, |> (nécessite R 4.1+)
useNativePipeOperatorNeverText=Jamais (utiliser le pipe de magrittr)
useNativePipeOperatorR41Text=Seulement avec R 4.1+
useNativePipeOperatorAlwaysText=Toujours
```

Reply:
I don't think we translate "pipe" in 🇫🇷, but we just pronounce it in English. cc @cderv

Reply:
There is this in `CmdConstants_fr.properties` : 

```
# insertPipeOperator
insertPipeOperatorLabel = Insérer un opérateur de tube
```

but I don't think I've heard people use `tube`.

Reply:
I agree @romainfrancois 

I don't think this would be `tube` - which is literally translation for pipe when doing plumbing. 
I would stick with `pipe` or use longer form  `l'opérateur pipe` to add context as done in `CmdConstants_fr.properties`.

Reply:
Thanks for confirming @cderv, updating the translations now